config/packages.tex

```diff
@@ -21,13 +21,13 @@
 
 \usepackage{caption}                    % didascalie
 
-\usepackage{chngpage,calc}              % centra il frontespizio
+\usepackage{changepage,calc}            % centra il frontespizio
 
 \usepackage{csquotes}                   % gestisce automaticamente i caratteri (")
 
 \usepackage{emptypage}                  % pagine vuote senza testatina e piede di pagina
 
-\usepackage{epigraph}			        % per epigrafi
+\usepackage{epigraph}			              % per epigrafi
```

Comment:
Se cambiamo i tab in spazi resta consistente su tutti gli editor

```suggestion
\usepackage{epigraph}                         % per epigrafi
```

Reply:
Convertiti tutti in teoria

Reply:
Questo su GitHub lo vedo ancora spostato in avanti

Reply:
Git non collabora e non vede le modifiche